plugins/a11y-keyboard/Makefile.am

```diff
@@ -42,6 +42,8 @@ liba11y_keyboard_la_SOURCES = 		\
 	msd-a11y-keyboard-plugin.c	\
 	msd-a11y-keyboard-manager.h	\
 	msd-a11y-keyboard-manager.c	\
+	msd-a11y-keyboard-atspi.h	\
+	msd-a11y-keyboard-atspi.c	\
```

Comment:
These files should not be included based on HAVE_LIBATSPI def, you must also remove the empty functions on msd-a11y-keyboard-atspi.c if the library is not available, as well as surrounding their usage in plugins/a11y-keyboard/msd-a11y-keyboard-manager.c and plugins/a11y-keyboard/msd-a11y-preferences-dialog.c, what's more you have to hide the setting in the preference dialog if the library is not available.

Reply:
The point in having them unconditionally was to make it easier on the manager.  I can indeed not build empty functions when not available if you prefer and add guards on the other end.

Reply:
FMPOV, it's better, otherwise you're making [software bloat](https://en.wikipedia.org/wiki/Software_bloat)

Reply:
I don't completely agree with software bloat here, although it's technically correct: there's another factor to consider, which is maintainability.  Code riddled with `#ifdef`s can quickly become a nightmare to navigate and understand, so sometimes it's better to keep the conditionals at a lower more abstract level.

Anyway, in this case I don't think it hurts readability and maintenance too much, so I did it.

So both concerns in your comment should be addressed now.